spacify-portal/src/main/java/org/uci/spacifyPortal/controllers/UserController.java

```diff
@@ -62,7 +67,8 @@ private List<RoomDetail> getOwnedRoomDetails(String userId) {
         return ownedRooms;
     }
 
-    private List<RoomDetail> getSubscribedRoomDetails(String userId) {
+    @GetMapping("/userSubscribedRooms/{userId}")
```

Comment:
is it working with Path Variable ? Like if we call the same function as a normal function not an API